app.js

```diff
@@ -904,42 +898,51 @@ async function recordChanges(command) {
         };
     }
 
-    const startDate = `${date1}T00:00:00.000Z`;
-    const endDate = `${date2}T23:59:59.999Z`;
+    const startDate = new Date(date1).toISOString();
+    const endDate = new Date(date2);
+    endDate.setUTCHours(23, 59, 59, 999);
+    const endDateString = endDate.toISOString();
 
     try {
         const res = await client.query(`
             SELECT * FROM "slughistory"
             WHERE changed_at >= $1 AND changed_at <= $2
             ORDER BY changed_at DESC;
-        `, [startDate, endDate]);
+        `, [startDate, endDateString]);
```

Comment:
can we please limit the query to 50 or 100 max results and warn user when it truncates?